src/fuzz/meson.build

```diff
@@ -56,8 +56,7 @@ fuzzers += [
          []],
 
         [['src/fuzz/fuzz-unit-file.c'],
-         [libcore,
-          libshared],
+         [libbasic, libsystemd_static, libcore, libshared_static],
```

Comment:
@yevgenypats I started to experiment with Fuzzit and it looks like fuzz targets should be linked statically against their dependencies. I'm wondering if there is a way to avoid this by sending `libshared` along with the fuzz target and pointing it to the library.

Reply:
Interestingly, even though the fuzzer failed to start with
```
INFO:root:/tmp/tmpwiylcb6d
INFO:root:Downloading fuzzer...
INFO:root:downloading orgs/X0gD7RDqzbNaqh4XUvUn/targets/qHXGJqipOXhjtWa8lAB1/jobs/qQfCjWEoEjrkrC940BMk/fuzzer.tar.gz
INFO:root:Downloading seed corpus...
INFO:root:downloading orgs/X0gD7RDqzbNaqh4XUvUn/targets/qHXGJqipOXhjtWa8lAB1/seed.tar.gz
INFO:root:artifact doesnt exist. skipping...
INFO:root:Syncing corpus from cloud
INFO:root:downloading orgs/X0gD7RDqzbNaqh4XUvUn/targets/qHXGJqipOXhjtWa8lAB1/corpus.tar.gz
INFO:root:artifact doesnt exist. skipping...
INFO:root:Running fuzzer for 1hr...
./fuzzer: error while loading shared libraries: libsystemd-shared-242.so: cannot open shared object file: No such file or directory
INFO:root:fuzzer exited with 127
INFO:root:exiting successfully...
```
Fuzzit reported that everything was OK: https://app.fuzzit.dev/admin/X0gD7RDqzbNaqh4XUvUn/target/qHXGJqipOXhjtWa8lAB1/qQfCjWEoEjrkrC940BMk.

Reply:
@evverx I Guess it's a bug. Yes I can see the error in the log you provided. it's possible to put the the shared.so in the zip but I need to add a feature so it will look for .SO in the same directory. I'll add this tomorrow and shoot you and update. We are in beta so there will be some bugs but we'll fix them quickly. Thanks for understanding:)

Reply:
In the meantime, I seem to have successfully sent one fuzz target to Fuzzit and judging by https://app.fuzzit.dev/admin/X0gD7RDqzbNaqh4XUvUn/target/8JwNICSj2oanzEjj39FQ/mdJVo2L3NX2IeUp7uQO5 it's being fuzzed but it was the second attempt :-) [The first one](https://app.fuzzit.dev/admin/X0gD7RDqzbNaqh4XUvUn/target/VDPYn5H8yXFQbCz5MMTT/snIpuoAnhPLNB47SAlmb) was seemingly successful too in the sense that the status was green but in fact I uploaded a zip archive (instead of `.tar.gz`) and the fuzzer kept failing with
```
INFO:root:/tmp/tmpri6wh2fv
INFO:root:Downloading fuzzer...
INFO:root:downloading orgs/X0gD7RDqzbNaqh4XUvUn/targets/VDPYn5H8yXFQbCz5MMTT/jobs/snIpuoAnhPLNB47SAlmb/fuzzer.tar.gz
INFO:root:Downloading seed corpus...
INFO:root:downloading orgs/X0gD7RDqzbNaqh4XUvUn/targets/VDPYn5H8yXFQbCz5MMTT/seed.tar.gz
Traceback (most recent call last):
  File "bot/main.py", line 428, in <module>
    main()
  File "bot/main.py", line 418, in main
    worker.fuzz()
  File "bot/main.py", line 310, in fuzz
    self._download_seed()
  File "bot/main.py", line 230, in _download_seed
    self._download_and_extract(f'{self._target_dir}/seed.tar.gz', 'seed', False)
  File "bot/main.py", line 205, in _download_and_extract
    with tarfile.open(gzip_file.name) as f:
  File "/usr/local/lib/python3.7/tarfile.py", line 1578, in open
    raise ReadError("file could not be opened successfully")
tarfile.ReadError: file could not be opened successfully
```
for about 15 minutes.

Reply:
> We are in beta so there will be some bugs but we'll fix them quickly. Thanks for understanding:)

@yevgenypats got it :-) By the way, what is the best way to report issues? I can keep reporting them here if it's OK but if there is a place to keep track of them I can switch to it.

Reply:
@evverx Thanks. Whatever works best for you. either here or you can report to me via email/slack

[https://join.slack.com/t/fuzzit/shared_invite/enQtNjI4MjA4NjY3MjE4LWY3MzViZmJkZmQ5ZDlhN2U5MDNlNjhiNjMxMmJhYzcxZTUzYjk4MDJiNjQzOGE2YWU1MzhkNTcwODI3MjQ3OTM](https://join.slack.com/t/fuzzit/shared_invite/enQtNjI4MjA4NjY3MjE4LWY3MzViZmJkZmQ5ZDlhN2U5MDNlNjhiNjMxMmJhYzcxZTUzYjk4MDJiNjQzOGE2YWU1MzhkNTcwODI3MjQ3OTM)